packages/autorest.go/.scripts/regeneration.js

```diff
@@ -128,6 +128,9 @@ generateFromReadme("armdataboxedge", databoxedge, 'package-2021-02-01', 'test/da
 const acr = './swagger/specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json';
 generate("azacr", acr, 'test/acr/azacr', '--module="azacr" --module-version=0.1.0 --openapi-type="data-plane" --rawjson-as-bytes --generate-fakes --azcore-version=1.9.0-beta.1');
 
+const acrDisallowUnknownFields = './swagger/specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json';
+generate("azacrdisallowunknown", acrDisallowUnknownFields, 'test/acr/azacr-disallow-unknown', '--module="azacr" --module-version=0.1.0 --openapi-type="data-plane" --rawjson-as-bytes --generate-fakes --disallow-unknown-fields --azcore-version=1.9.0-beta.1');
```

Comment:
tried to use this to generate models with --disallow-unknown-fields, but could not get this script to work

Reply:
Few things.
`--module="azacrdisallowunknown"`
The `--azcore-version` can be removed as `v1.9.0` is now the default version.
If you `rush regenerate` it should work. Or did you try that and it failed?